Dexterous-Rambutan/battle-code
# Adds  basic implementation of solution tester worker processes

#### What's this PR do?
- Adds worker processes for queueing up submitted solutions in Redis
#### What are the important parts of the code?
- \server\public\addProblemSolution.html - form to add toy problems or solutions to database
- \worker - code to create worker process to solve solution
#### How should this be tested by the reviewer?
- start redis, postgres, node server.js, node worker\solutionTester.js
- use http://localhost/addProblemSolution.html to add a toy problem and test cases (e.g. nthFibonacci)
- use the same website to add test solution code
- successful solutions should see "victory!" logged in the solutionTester.js terminal
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

ref #28 
#### Screenshots (if appropriate)
